.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
-python:
-  - "2.7"
-install: "python setup.py install"
-script: "py.test tests/unit"
 branches:
     only:
         - master
         - dogfood
+services:
+  - postgres
+python:
+  - "2.7"
+env:
+  - DB=sqlite
+  - DB=postgres
+install:
+  - sh -c "if [ $DB = postgres ]; then pip install psycopg2; fi"
+  - python setup.py install
+before_script:
+  - sh -c "if [ $DB = postgres ]; then psql -c 'CREATE DATABASE haas_travis;' -U postgres; fi"
+  - sh -c "if [ $DB = postgres ]; then cp ci/testsuite.cfg.postgres testsuite.cfg; fi"
+script: "py.test tests/unit tests/stress.py"
```

Comment:
I added `-n auto` to travis for sqlite only. I think writing the logic to distribute the tests amongst multiple postgres databases is going to be nontrivial, enough so that it probably should be its own issue.